spec/requests/api/job_definitions_spec.rb

```diff
@@ -0,0 +1,135 @@
+require 'rails_helper'
+
+describe 'job_definitions' do
+  let(:service_name) { :test_client_name }
+  let(:secret_key) { 'secret_key' }
+  let(:env) do
+    {
+      accept: 'application/json',
+      authorization: "Basic #{Base64.encode64("#{service_name}:#{secret_key}")}",
+    }
+  end
+  let(:result) { JSON.parse(response.body) }
+
+  describe 'GET /v1/definitions' do
+    let(:definition) do
+      create(:job_definition, script: 'noop:', api_allowed: true)
+    end
+
+    before do
+      definition
+    end
+
+    it 'returns definitions' do
+      get "/v1/definitions", params: {}, env: env
+      expect(result).to eq(
+        [{
+          'id' => definition.id,
+          'name' => definition.name,
+          'description' => definition.description,
+          'script' => definition.script
+        }]
+      )
+      expect(response.status).to eq(200)
+    end
+  end
+
+  describe 'POST /v1/definitions' do
```

Comment:
I want to tests with invalid parameters.

Reply:
I have fixed it in 939b69d.